### ISSUE TITLE
Issue #111

### ISSUE THREAD

**Issue by Crazyx001** [mono/monotorrent#114]
Hello. Sorry for such a long answer an issue #111, because I was away on business. Unfortunately your advice didn’t help me and yes, I am using the latest library build.

**Comment by alanmcgovern** [comment]
Can you share the torrent with me to help diagnose the issue?

**Comment by Crazyx001** [comment]
Yes. Here is a torrent that does not download.
[MyTor.zip](https://github.com/mono/monotorrent/files/3528526/MyTor.zip)

**Comment by alanmcgovern** [comment]
I loaded it up and it seems to work just fine. The only issue I can see is that there is only 1 other peer running this torrent.

I loaded it up in uTorrent at the same time to see if uTorrent had more success, but it didn't even connect to the peer that MonoTorrent was able to connect to. The only client uTorrent connected to was the local MonoTorrent client running on the same machine.

<img width="1384" alt="Screenshot 2019-08-22 at 10 02 53" src="https://user-images.githubusercontent.com/264396/63501838-41516c00-c4c4-11e9-8569-fa5de411d045.png">

What issue are you seeing? Is it just that there are very few peers available, and perhaps you tried to download it at a time when the other peer I can find was not online, or was not findable via a tracker or DHT?
